Makefile

```diff
@@ -92,19 +92,23 @@ integration: build
 	@echo "SOCI_SNAPSHOTTER_PROJECT_ROOT=$(SOCI_SNAPSHOTTER_PROJECT_ROOT)"
 	@GO111MODULE=$(GO111MODULE_VALUE) SOCI_SNAPSHOTTER_PROJECT_ROOT=$(SOCI_SNAPSHOTTER_PROJECT_ROOT) ENABLE_INTEGRATION_TEST=true go test $(GO_TEST_FLAGS) -v -timeout=0 ./integration
 
-benchmarks:
-	@echo "$@"
-	@cd benchmark/performanceTest ; GO111MODULE=$(GO111MODULE_VALUE) go build -o ../bin/PerfTests . && sudo ../bin/PerfTests
-	@cd benchmark/comparisonTest ;  GO111MODULE=$(GO111MODULE_VALUE) go build -o ../bin/CompTests . && sudo ../bin/CompTests
+benchmarks: benchmarks-perf-test benchmarks-comparison-test
+
+build-benchmarks: benchmark/bin/PerfTests benchmark/bin/CompTests
+
+benchmark/bin/PerfTests: FORCE
```

Comment:
nit: Any reason not to add this to phony targets? I presume it's because it's never intended to be directly called and thus helps create a separation of ideas in that sense?

Reply:
It's not _technically_ a phony target - it really does produce this file. I think there's a question on whether we should consider our build targets to be phone and let go handle incremental builds (which is effectively what we're doing now), or actually have the targets depend on the go source files. 

I'm fine with either, I just want it all to be consistent and right now that means non-phony using `FORCE`.